src/Cake.Common/Tools/ReportGenerator/ReportGeneratorReportType.cs

```diff
@@ -72,7 +72,7 @@ public enum ReportGeneratorReportType
         /// <summary>
         /// Same as HTMLInline but with modified CSS that matches the look and feel of Azure Pipelines.
         /// </summary>
-        HtmlInlineAzurePipelines = 13,
+        HtmlInline_AzurePipelines = 13,
```

Comment:
This value has already shipped with Cake so would be a breaking change.

Reply:
Yeah.. this is tricky. I don't know why it was changed in https://github.com/cake-build/cake/commit/d73c086976d92ab1e7ceffd61a35c1026d18d2b4, but the correct parameter to pass to ReportGenerator.exe has the underscore in it.

Has this actually shipped though? The last release v0.32.1 was on Jan 4, and that breaking commit was made on Jan 14, so if this is merged before the next release then everything would be ok?

Reply:
Agreed, this hasn't shipped yet.

Reply:
Flag has shipped but had wrong number. This is in common and if tool expects with underscore this is correct. Agree this should break and mark this as resolved.